src/parser.mjs

```diff
@@ -613,9 +613,11 @@ export class ExifParser extends Reader {
 		if (this.options.postProcess) {
 			let start = this.xmp.indexOf('<x:xmpmeta')
 			let end = this.xmp.indexOf('x:xmpmeta>') + 10
-			this.xmp = this.xmp.slice(start, end)
+			if (start >= 0 && end > 0) {
```

Comment:
This is a fix for XMP strings that don't use the `x:xmpmeta` tag, like the ones in the FLIR Duo Pro photos.